public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ConfigEditor.tsx

```diff
@@ -138,12 +141,18 @@ export class ConfigEditor extends PureComponent<Props, State> {
           getWorkspaces={this.getWorkspaces}
         />
 
-        <InsightsConfig
-          options={options}
-          onUpdateJsonDataOption={this.onUpdateJsonDataOption}
-          onUpdateSecureJsonDataOption={this.onUpdateSecureJsonDataOption}
-          onResetOptionKey={this.resetSecureKey}
-        />
+        {isInsightsConfigured ? (
+          <InsightsConfig
+            options={options}
+            onUpdateJsonDataOption={this.onUpdateJsonDataOption}
+            onUpdateSecureJsonDataOption={this.onUpdateSecureJsonDataOption}
+            onResetOptionKey={this.resetSecureKey}
+          />
+        ) : (
+          <Alert severity="info" title="Application Insights credentials are no longer supported">
+            Configure using Azure AD App Registration above and query Application Insights using Metrics or Logs.
+          </Alert>
```

Comment:
mmh, shouldn't we show this alert if insights are configured? for new users, that never used app insights, we can just hide the section, can't we?

Reply:
Yup lets scrap this